test/gst-msdk/encode/av1.py

```diff
@@ -57,6 +58,13 @@ def test(self, case, gop, bframes, tilecols, tilerows, qp, quality, profile):
     self.init(spec, case, gop, bframes, tilecols, tilerows, qp, quality, profile)
     self.encode()
 
+  @slash.parametrize(*gen_av1_cqp_lp_parameters(spec_r2r))
+  def test_r2r(self, case, gop, bframes, tilecols, tilerows, qp, quality, profile):
+    self.init(spec, case, gop, bframes, tilecols, tilerows, qp, quality, profile)
```

Comment:
`spec` -> `spec_r2r`

Reply:
thanks review and catch this issue, updated